ui/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -468,33 +472,25 @@ export default class AdvancedTab extends PureComponent {
       userHasALedgerAccount,
     } = this.props;
 
-    const LEDGER_TRANSPORT_NAMES = {
-      LIVE: t('ledgerLive'),
-      WEBHID: t('webhid'),
-      U2F: t('u2f'),
+    const TRANSPORT_NAMES = {
+      [LedgerTransportTypes.live]: t('ledgerLive'),
+      [LedgerTransportTypes.webhid]: t('webhid'),
+      [LedgerTransportTypes.u2f]: t('u2f'),
     };
 
     const transportTypeOptions = [
-      {
-        name: LEDGER_TRANSPORT_NAMES.LIVE,
-        value: LedgerTransportTypes.live,
-      },
-      {
-        name: LEDGER_TRANSPORT_NAMES.U2F,
-        value: LedgerTransportTypes.u2f,
-      },
+      getLedgerTransportOption(LedgerTransportTypes.live, TRANSPORT_NAMES),
+      getLedgerTransportOption(LedgerTransportTypes.u2f, TRANSPORT_NAMES),
     ];
 
     if (window.navigator.hid) {
```

Comment:
This `window.navigator.hid` reference occurs client-side, so no need for the `isManifestv3` flag here